### PR TITLE
Int 1049 fix quick filter values

### PR DESCRIPTION
**Pull request by bryanateoan**
# Overview
Teamserver expects the quickfilter values to be SCREAMING_SNAKE_CASE

# What changed
1. updated the quickfilter values to be SCREAMING_SNAKE_CASE
2. Added test for filter_org_traces
3. updated README and test config example

### CODE REVIEW

**Review by arturoespinoza15 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -57,3 +57,9 @@ Then run tests with nose:
 ```commandline
     nosetests
 ```
+
+You can run nosetests with python3 using:
+
+```commandline
+    python3 -m "nose" [test file]
```
Comment (cgperez) [inline README.md:64]: could you mention that you need to add `test/...` before the test file name

**Review by cgperez — APPROVED** [review]
✅ verified

<pre>
❯ python3 -m "nose" tests/trace_api_test.py
..........
----------------------------------------------------------------------
Ran 10 tests in 0.364s

OK
</pre>

File: tests/trace_api_test.py [hunk 2]
```diff
@@ -55,3 +55,14 @@ def get_total_trace_trend_test(self):
 
     def get_trace_time_to_remediate_month_trend_test(self):
         self.assertEqual(200, self.sdk.get_trace_time_to_remediate_month_trend(self.org_uuid).status_code)
+
+    def trace_quick_filter_enum_open_is_correct(self):
```
[the remark this answers is not in the document]
Comment (bryanateoan) [reply tests/trace_api_test.py]: moved to a separate file

File: README.md (hunk 1 above)
Comment (bryanateoan) [reply README.md]: added clarification

**Pull request merged**